engine-tests/src/main/java/org/terasology/DisplayEnvironment.java

```diff
@@ -83,19 +85,28 @@ protected void setupDisplay() {
         LWJGLHelper.initNativeLibs();
 
         try {
-            Display.setDisplayMode(new DisplayMode(1, 1));
+            Display.setDisplayMode(new DisplayMode(320, 240));
             Display.create(context.get(Config.class).getRendering().getPixelFormat());
         } catch (LWJGLException e) {
             throw new IllegalStateException(e);
         }
+
+        context.put(ShaderManager.class, new ShaderManagerLwjgl());
     }
 
     @Override
     protected void setupAssetManager() {
-        AssetManager assetManager = new AssetManagerImpl(context.get(ModuleManager.class).getEnvironment());
+        NullAudioManager audioManager = new NullAudioManager();
+        context.put(AudioManager.class, audioManager);
```

Comment:
Can these two lines be moved to the end of the method, just before the two setAssetFactory calls are made?